src/mixins/add-edit-org-person-mixin.ts

```diff
@@ -205,11 +210,36 @@ export default class AddEditOrgPersonMixin extends Vue {
   // *** TODO: test this (should be disabled if it's the only role displayed)
   get disableDirectorRole (): boolean {
     // always enable this role for restoration filings
-    if (this.isRestorationFiling) return false
+    if (this.isFullRestorationFiling || this.isLimitedRestorationFiling) return false
     // disable this role if it's the only role displayed
     return (!this.showCompletingPartyRole && !this.showIncorporatorRole && this.showDirectorRole)
   }
 
+  /** Whether the Heir or Legal Representative role should be disabled. */
+  get disableHeirLegalRepRole (): boolean {
+    return false
+  }
```

Comment:
I added these getters, even though their return value is static, to keep the template consistent and for future use.